static/templates/settings/playground_settings_admin.hbs

```diff
@@ -0,0 +1,21 @@
+<div id="playground-settings" class="settings-section" data-name="playground-settings">
+    <div class="admin-table-wrapper">
+        <p>
+            {{#tr}}
+            Configure playground options to open code appearing in message code blocks
+            in these external playgrounds.
```

Comment:
Perhaps: 
"Configure external sites that can be used to preview or run content present in a code block."